config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -897,7 +902,6 @@ periodics:
     preset-service-account: "true"
     preset-azure-cred: "true"
     preset-azure-windows: "true"
-    preset-windows-repo-list-master: "true"
```

Comment:
the repolists are not needed for the azure disk/file prs?

Reply:
No, they do not use it at all. Just piggy riding this PR to clean them up.